sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -463,6 +512,12 @@ def replace_container(  # pylint:disable=docstring-missing-param
         default_ttl: Optional[int] = None,
         conflict_resolution_policy: Optional[Dict[str, Any]] = None,
         populate_query_metrics: Optional[bool] = None,
+        *,
+        session_token: Optional[str] = None,
+        initial_headers: Optional[Dict[str, str]] = None,
+        etag: Optional[str] = None,
+        match_condition: Optional[MatchConditions] = None,
+        analytical_storage_ttl: Optional[int] = None,
```

Comment:
Could we move the docstring for this param so that it's with the other "keywords" and "raises" is at the bottom? Not sure if that would render correctly...

Reply:
big oops on our end, went ahead and changed it for the async one as well